site/gdocs/components/Chart.tsx

```diff
@@ -22,6 +22,7 @@ export default function Chart({
 }) {
     const { isPreviewing, archiveContext } = useContext(DocumentContext)
     const refChartContainer = useRef<HTMLDivElement>(null)
+    const archiveIframeRef = useRef<HTMLIFrameElement | null>(null)
     useEmbedChart(0, refChartContainer, isPreviewing)
```

Comment:
I don't think we'll wanna run `useEmbedChart` on an archived chart (which we're now doing, because `refChartContainer` is not null anymore)?
It'll run the MultiEmbedder on these charts